### ISSUE TITLE
Exception thrown when clicking on a scene

### ISSUE THREAD

**Issue by watermelonwolverine** [unsoluble/smalltime#153]
Exception thrown when clicking on a scene.

Steps to reproduce:
1. Click on a scene
2. Maybe even open configuration

Expectation:
No exception thrown

Result:
```
Uncaught (in promise) Error: This is not a registered game setting
[Detected 3 packages: force-client-settings, lib-wrapper, smalltime]
    at ClientSettings.get (foundry.js:2758:42)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:507:22)
    at ClientSettings.settingsGet (force-client-settings.js:146:12)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:616:16)
    at 🎁ClientSettings.prototype.get#0 (libWrapper-wrapper.js:189:20)
    at Object.fn (smalltime.js:529:47)
    at #call (foundry.js:724:20)
    at Hooks.callAll (foundry.js:681:17)
    at SceneConfig._render (foundry.js:5112:13)
    at async SceneConfig._render (foundry.js:5780:5)

  | 🎁call_wrapped | @ | libWrapper-wrapper.js:507
  | settingsGet | @ | force-client-settings.js:146
  | 🎁call_wrapper | @ | libWrapper-wrapper.js:616
  | 🎁ClientSettings.prototype.get#0 | @ | libWrapper-wrapper.js:189
  | (anonymous) | @ | smalltime.js:529
  | #call | @ | foundry.js:724
  | callAll | @ | foundry.js:681
  | _render | @ | foundry.js:5112
  | await in _render (async) |   |  
  | _render | @ | foundry.js:5780
  | render | @ | foundry.js:5026
  | render | @ | foundry.js:6352
  | render | @ | foundry.js:65896
  | _onClickDocumentName | @ | foundry.js:60364
  | dispatch | @ | jquery.min.js:2
  | y.handle | @ | jquery.min.js:2
```

Settings a break point at foundry.js:2758 delivers this:
```
{
    "namespace": "pf2e",
    "key": "pf2e.automation.rulesBasedVision"
}
```

Report-Generator output:
```
```Versions:
  Foundry: 10.287
  System: dnd5e version 2.0.3
User:
  Role: GAMEMASTER
Settings:
  Disable_Canvas: Disabled
  Max_FPS: 60
  Token_Drag_Vision: Disabled
  Performance_Mode: 2
  Token_Vision_Animation: Enabled
  Light_Source_Animation: Enabled
  Zoomed_Texture_Antialiasing: Enabled
  Animate_Roll_Tables: Enabled
Scene:
  Walls: 150
  Lights: 9
  Tokens: 10
  Tiles: undefined
  Sounds: 0
  Drawings: 0
  Notes: 0
  Dimensions: 19200 x 15360
  Background: undefined x undefined
  Foreground: undefined x undefined
WebGL:
  Context: WebGL2RenderingContext
  GL_Vendor: WebKit
  Renderer: WebKit WebGL
  Unmasked_Renderer: ANGLE (NVIDIA, NVIDIA GeForce GTX 960M Direct3D11 vs_5_0 ps_5_0, D3D11)
  WebGL_Version: WebGL 2.0 (OpenGL ES 3.0 Chromium)
  MAX_TEXTURE_SIZE: 16384
  MAX_RENDERBUFFER: 16384
Database:
  Actors: 42
  Items: 3
  Scenes: 23
  Journals: 260
  Tables: 9
  Chat: 0
  Macros: 132
Data_Sizes:
  Actors: 1.54 MB
  Items: 6.7 KB
  Scenes: 351.77 KB
  Journals: 520.9 KB
  Tables: 10.15 KB
  Chat: 2 Bytes
  Macros: 199.28 KB
Browser:
  Platform: Win32
  Vendor: Google Inc.
  Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
Memory:
  Heap_Limit: 4 GB
  Heap_Total: 277.8 MB
  Heap_Used: 253.08 MB
Load:
  World_Size: 2.6 MB
  Scene_Size: 8.13 MB
Modules:
  Total: 62
  Enabled: 52
Active_Modules:
  about-time: About Time v10.0.1
  advanced-drawing-tools: Advanced Drawing Tools v2.0.2
  advanced-macros: Advanced Macros vv1.17
  autoanimations: Automated Animations v3.3.16
  betterroofs: Better Roofs v1.7.3
  combatbooster: Combat Booster: Turn Marker, Recent Actions and more v3.2
  compendium-folders: Compendium Folders v2.5.3
  dae: Dynamic effects using Active Effects v10.0.9
  ddb-importer: D&D Beyond Importer v3.1.13
  ddb-importer-compendiums: DDB Importer Compendiums v1.0.0
  debug-report: Debug Report Generator v0.1.5
  df-manual-rolls: DF Manual Rolls v2.3.1
  df-settings-clarity: DF Settings Clarity v3.2.1
  dfreds-convenient-effects: DFreds Convenient Effects v3.0.1
  dice-calculator: Dice Tray v1.5.2
  dice-so-nice: Dice So Nice! v4.6.2
  dice-tooltip: Dice Tooltip - DnD5e v2.3.1
  disable-mouse-wheel-sliders: Disable Mouse Wheel Control for Sliders v1.2.5
  force-client-controls: Force Client Controls v1.0.4
  force-client-settings: Force Client Settings v2.3.2
  forien-quest-log: Forien's Quest Log v0.7.11
  foundryvtt-simple-calendar: Simple Calendar v2.1.27
  fxmaster: FXMaster v3.5.0
  itemacro: Item Macro v1.8.0
  jb2a_patreon: JB2A - Patreon Complete Collection v0.4.7
  levels: Levels v3.2.7
  lib-changelogs: Changelogs & Conflicts v0.7.7
  lib-wrapper: libWrapper v1.12.10.0
  lmrtfy: Let Me Roll That For You! v3.0.3
  mcc: Module Compatibility Checker v1.3.2
  midi-qol: Midi QOL v10.0.14
  multi-token-edit: Mass Edit v1.20.0
  pause-icon: Alternative Pause Icon v1.0.3
  perfect-vision: Perfect Vision v4.0.26
  permission_viewer: Ownership Viewer v0.10.0
  popout: PopOut! v2.9
  select-tool-everywhere: Select tool everywhere v1.4.0
  sequencer: Sequencer v2.3.8
  shared-data: Shared Data v1.0.0
  smalltime: SmallTime v1.15.1
  smarttarget: Smart Target v0.9.3
  socketlib: socketlib v1.0.12
  tidy5e-sheet: Tidy5e Sheet v0.6.6
  times-up: Times Up v10.0.2
  token-animation-tools: Token Animation Tools v1.3.1.0
  token-attacher: Token Attacher v4.5.2
  token-chat-link: Token Chat Link v1.1.0
  token-variants: Token Variant Art v4.3.0
  token-z: Token Z v0.7.6
  tokenmagic: Token Magic FX v0.6.1.2
  vtta-tokenizer: Tokenizer v3.9.5
  wall-height: Wall Height v4.7.8
```
```

**Comment by unsoluble** [comment]
Curious. That doesn't happen for me in any of my 5e worlds. Will investigate.

**Comment by unsoluble** [comment]
Ah, okay, got it. Sec.

**Comment by unsoluble** [comment]
Should be fixed in 1.15.2 — lemme know if not.

**Comment by unsoluble** [comment]
Oops I meant 1.15.3. _whistles innocently_
